src/datapoints-graph/charts/echarts-options.service.ts

```diff
@@ -13,6 +13,9 @@ import { YAxisService } from './y-axis.service';
 import { ChartTypesService } from './chart-types.service';
 import type { TooltipFormatterCallback } from 'echarts/types/src/util/types';
 import type { TopLevelFormatterParams } from 'echarts/types/src/component/tooltip/TooltipModel';
+import { AlarmStatus, IAlarm, IEvent } from '@c8y/client';
+import { ICONS_MAP } from './svg-icons.model';
+import { CustomSeriesOptions } from './chart.model';
 
 @Injectable()
 export class EchartsOptionsService {
```

Comment:
comment to whole file - methods that are not exposed and used outside service can be private ones.